packages/core/src/sharedWishlists/redux/selectors.js

```diff
@@ -187,30 +187,30 @@ export const getSharedWishlistItem = createSelector(
  *
  * @param {object} state - Application state.
  * @param {number} sharedWishlistId - Numeric identifier of the shared wishlist.
+ * @param _
```

Comment:
Same here

Reply:
Yes, JSDoc was complaining about it. I even found another case of this in the getCheckoutDeliveryBundleUpgrade selector